1-exercises/A-reading-list/script.js

```diff
@@ -1,7 +1,27 @@
 function readingList(books) {
-  // Write your code here...
+let divEl =document.getElementById("content");
+books.forEach((book) => {
+ let pEl = document.createElement("p");
+ let liEl  = document.createElement("li");
+ let imgEl  = document.createElement("img");
+ let ulEl = document.createElement("ul");
+ 
+    pEl.appendChild(
+      document.createTextNode(`"${book.title}" by ${book.author}`)
+    );
+    liEl.appendChild(pEl);
+    imgEl.setAttribute("src", book.bookCoverImage);
+    liEl.appendChild(imgEl);
+    ulEl.appendChild(liEl); 
+
+        if (book.alreadyRead) liEl.className = "green";
+    else liEl.className = "red";
```

Comment:
your red class for unread books works well, however you don't see to have created a class called `green` so the read books are not currently getting a background colour set